screens/Albums.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { globalStyles } from '../styles/global';
+import Card from '../components/Card';
+import { FlatList } from 'react-native-gesture-handler';
```

Comment:
This is why I like using the Webstorm IDE haha, so sometimes you'll find that the automated imports pull from whatever they think you want to use. That being said, while the FlatList is a part of the `react-native-gesture-handler` package, more likely you meant to grab the component from `react-native`

```suggestion
import { StyleSheet, Text, TouchableOpacity, View, FlatList } from 'react-native';
import { globalStyles } from '../styles/global';
import Card from '../components/Card';
```

Reply:
Thanks, I did notice this happened a couple of times and I fixed a few, must have missed this one. Looks like I'll have to look into getting Webstorm. Does Webstorm IDE help with identifying the proper import package?